astropy/visualization/wcsaxes/helpers.py

```diff
@@ -24,6 +28,56 @@
 }
 
 
+def add_compass(
+    ax,
+    north_angle,
+    length=0.15,
+    sep_N=-0.1,
+    sep_E=0.04,
+    color="white",
+    **kwargs,
+):
```

Comment:
To match ``add_beam`` and ``add_scalebar``, could you add the following options:

```
    corner="top left",
    frame=False,
    borderpad=0.4,
```

along with their implementation? Currently there is no easy way to customize the position of the compass.

Reply:
I am doing this as suggested, but just a question so I learn best practices. Frameon, pad, and borderpad are all arguments to AnchoredDirectionArrow, so wouldn't this be just as straightforward to do without explicitly naming them as options here?  Is the point that you'd have to go down a rabbit-hole of inheritances to see that these are options?